tests/unit/commands/doctor-test.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
```

Comment:
Can we be consistent with the space after `'use strict';`. I prefer the space, but depends on what is being done elsewhere.